src/cf-forms/usage.md

```diff
@@ -324,10 +337,151 @@ Provides sizeable margins between groups of form elements.
 <div class="btn-inside-input">
     <input class="input__super"
            type="text"
-           value="This is some really long text to make sure that the button doesn't overlap the content in such a way that this input becomes unusable." title="Test input">
+           value="This is some really long text to make sure that the button doesn't overlap the content in such a way that this input becomes unusable."
+           title="Test input">
     <button class="btn btn__super btn__link btn__secondary">
         Clear
         <span class="cf-icon cf-icon-delete"></span>
     </button>
 </div>
 ```
+
+### Checkboxes with large target areas
+
+<fieldset class="u-reset">
+    <div class="form-l_col
+                form-l_col-1">
+        <legend class="form-label-header">
+            Pets you own
+        </legend>
+    </div>
+    <div class="form-l_col
+                form-l_col-1-2
+                form-l-col__inset">
+        <div class="form-l-inset_container">
+            <input class="cf-input"
+                   type="checkbox"
+                   name="form_id"
+                   id="input_id_cats">
+            <label class="cf-input_label"
+                   for="input_id_cats">
+                Cats
+            </label>
+        </div>
+    </div>
+    <div class="form-l_col
+                form-l_col-1-2
+                form-l-inset">
+        <div class="form-l-inset_container">
+            <input class="cf-input"
+                   type="checkbox"
+                   name="form_id"
+                   id="input_id_dogs">
+            <label class="cf-input_label"
+                   for="input_id_dogs">
+                Dogs
+            </label>
+        </div>
+    </div>
+</fieldset>
+
+```
+
```

Comment:
Sorry, you have markup on L:351-386 which will be rendered. Add the same markup to the code blocks so folks know what it is. The same is happening below, but there's no empty code block like you have here.